cfgov/v1/util/ref.py

```diff
@@ -183,6 +183,13 @@
             ("student-loans", "Student loans"),
         ),
     ),
+    (
+        "Petition",
+        (
+            ("open", "Open petition"),
+            ("closed", "Closed petition"),
+        ),
+    ),
 ]
```

Comment:
```suggestion
```

Did you accidentally bring in #7908 here?

Reply:
whoops